File: API/XApp6/XApp6/XApp6/Models/Evento.cs
using System;
namespace XApp6.Models
{
    public class Evento
    {
		public int id { get; set; }
		public string nome { get; set; }
        public string tipo { get; set; }
        public string local { get; set; }
        public List<Curso> cursos { get; set; }
        public List<Disciplinas> disciplina { get; set; }
    }
}

File: esaelaravelmarco/app/Evento.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use App\Curso;
use App\Disciplina;

class Evento extends Model
{
	protected $table = "eventos";
	protected $primaryKey = "id";
	protected $fillable = array("nome", "tipo","local");
	public $timestamps = true;
	
	public function cursos()
	{
		return $this->belongsToMany('App\Curso', 'curso_evento')->withTimestamps();
	}
	
	public function disciplinas()
	{
		return $this->belongsToMany('App\Disciplinas', 'disciplina_evento')->withTimestamps();
	}
	
}
